lib/mix/tasks/new.ex

```diff
@@ -19,67 +15,79 @@ defmodule Mix.Tasks.Scenic.New do
   mix archive.install hex scenic_new
   ```
 
-  ## Build the Starter App
+  ## Build the Starter Application
 
-
-  First, navigate the command-line to the directory where you want to create your new Scenic app. Then run the following commands:  (change `my_app` to the name of your app...)
+  First, navigate the command-line to the directory where you want to create
+  your new Scenic application. Then run the following commands: (change
+  `my_app` to the name of your application)
 
   ```bash
   mix scenic.new my_app
   cd my_app
   mix do deps.get, scenic.run
   ```
 
-
   ## Running and Debugging
 
-  Once the app and its dependencies are set up, there are two main ways to run it.
+  Once the application and its dependencies are set up, there are two main ways
+  to run it.
 
-  If you want to run your app under IEx so that you can debug it, simply run
+  If you want to run your application under `IEx` so that you can debug it,
+  simply run
 
   ```bash
   iex -S mix
   ```
 
   This works just like any other Elixir application.
 
-  If you want to run your app outside of iex, you should start it like this:
+  If you want to run your application outside of `IEx`, you should start it
+  like this:
 
   ```bash
   mix scenic.run
   ```
 
-  ## The Starter App
+  ## The Starter Application
 
-  The starter app created by the generator above shows the basics of building a Scenic application. It has four scenes, two components, and a simulated sensor.
+  The starter application created by the generator above shows the basics of
+  building a Scenic application. It has four scenes, two components, and a
+  simulated sensor.
 
   Scene | Description
   --- | ---
-  Splash | The Splash scene is configured to run when the app is started in the `config/config.exs` file. It runs a simple animation, then transitions to the Sensor scene. It also shows how intercept basic user input to exit the scene early.
+  Splash | The Splash scene is configured to run when the application is started in the `config/config.exs` file. It runs a simple animation, then transitions to the Sensor scene. It also shows how intercept basic user input to exit the scene early.
```

Comment:
These tables look good in the HTML version, but it does not look well when you use `mix help scenic.new` or the `h` helper inside `IEx`:

![screen shot 2018-09-16 at 10 53 26 am](https://user-images.githubusercontent.com/34700/45598287-151d9900-b99f-11e8-85fe-6232f86ab5a4.png)

@boydm My suggestion is to change the format here, but I want to know your opinion first,

Reply:
Really good point. I was using tables in several places in the Scenic docs, but have been ripping them out in favor of plain old lists. Doesn't look as nice in html, but is better overall.

Reply:
This is why I changed the documentation I touched so far to be just a list instead of table view.